src/modules/communication/internals/twilio-service.ts

```diff
@@ -35,3 +39,7 @@ export default class TwilioService {
     }
   }
 }
+
+export default ConfigService.getBoolValue('communication.mock')
```

Comment:
This is still adding the mock if else in production code.

Is it not possible use sinon in scenario testing to mock the third party api call?